vignettes/secuTrialR-package-vignette.Rmd

```diff
@@ -1,172 +1,570 @@
+Generally, it is advisable to recalculate score variables before data analysis.
+A rule of thumb: The more complex a score is and the more data from different 
+forms is necessary for its calculation the more likely its value should be recalculated.
+The below function will allow you to detect which variables this concerns.
+\vspace{5pt}
 
 ```{r}
-val_ovv_location <- system.file("extdata", "sT_exports", "BMD",
-                                "bmd_validation_overview.xlsx", 
-                                package = "secuTrialR")
+return_scores(ctu05_data)
+```
+
+\newpage
+
+## Finding changes/differences in exports
+
+In ongoing studies it is possible that changes to the secuTrial data entry
+interface (i.e. the electronic case report forms) are made. Sometimes these
+changes may call for adjustments in analysis code. It is considered good practice
+to run  `diff_secuTrial()` on the last export and the current export of a project
+to at least make yourself aware of potential changes in the setup. If there are
+differences, the results of this function should be interpreted as a first indicator
+since they may not cover all alterations. Information is returned on new forms and
+variables. A detailed list of changes can be produced in the secuTrial
+FormBuilder with "Compare project setup".
+\vspace{5pt}
+
+``` {r}
+# load second export from the same project
+export_location <- system.file("extdata", "sT_exports", "longnames",
+                               "s_export_CSV-xls_CTU05_long_ref_miss_en_utf8.zip",
+                               package = "secuTrialR")
+# read all export data
+ctu05_data_new <- read_secuTrial_raw(data_dir = export_location)
+
+# show diff
+diff_secuTrial(ctu05_data, ctu05_data_new)
+
 ```
 
-In order to asses variable completeness we first require a loaded secuTrial
-validation overview and a loaded secuTrial export. Both the validation
-overview and the data export should be from the same time (not more than minutes
-apart if possible) to ensure data integrity.
+This `list` contains only empty entries. Thus, you can conclude that nothing has changed.
 
-The secuTrial export has already been loaded above. The code below shows how to load the validation overview.
-The validation overview contains a line for every variable with a missing or erroneous value. As explained 
-earlier, patient 999 was added with missing values. As you can see the validation report only contains entries
-for "Patient" 999.
+## Conversion to SPSS, STATA, SAS
+
+Given that you are working with R it is unlikely that you need such conversions for
+yourself. However, collaborators may ask for data which is readily importable into SPSS,
+STATA or SAS. For this you
+can use `write_secuTrial()`.
+\vspace{5pt}
+
+```{r, eval = FALSE}
+# retrieve path to a temporary directory
+tdir <- tempdir()
+# write spss
+write_secuTrial(ctu05_data, format = "sav", path = tdir)
+```
+
+Since this has not been heavily tested or used there may be issues and
```

Comment:
Transfer these last two lines of the section to immediately precede the example (some people might oversee it otherwise). Also, I'm not sure we should export an experimental function that we do not feel confident about... Maybe we can ask some SPSS /STATA /SAS users to do some user testing?

Reply:
@aghaynes what do you think. I think I agree with @markomi